fennel/datasets/datasets.py

```diff
@@ -1956,6 +1994,29 @@ def visitDrop(self, obj) -> DSSchema:
         output_schema.name = output_schema_name
         return output_schema
 
+    def visitDropNull(self, obj):
+        input_schema = copy.deepcopy(self.visit(obj.node))
+        output_schema_name = f"'[Pipeline:{self.pipeline_name}]->dropnull node'"
+        if obj.columns is None or len(obj.columns) == 0:
```

Comment:
We should allow dropnull on the entire dataset too? Which would basically create dropnulls for any optional column in the dataset, if there are no optional columns then we can throw an error